game/world/managers/objects/spell/CastingSpell.py

```diff
@@ -140,6 +140,14 @@ def is_refreshment_spell(self):
 
         return has_sitting_attribute and is_regen_buff
 
+    def is_summon_spell(self):
+        if len(self.effects) == 0:
+            return False
```

Comment:
Isn't this a bit redundant? It won't enter in the for loop below lf the length is 0 anyway.

Reply:
Yes.